reference-architecture/aws-ansible/ose-on-aws.py

```diff
@@ -70,7 +70,10 @@
 @click.option('--containerized', default='False', help='Containerized installation of OpenShift',
               show_default=True)
 @click.option('--s3-bucket-name', help='Bucket name for S3 for registry')
-@click.option('--s3-username', help='S3 user for registry access')
+@click.option('--github-client-id', help='GitHub OAuth ClientID')
+@click.option('--github-client-secret', help='GitHub OAuth Client Secret')
+@click.option('--github-organization', multiple=True, help='GitHub Organization')
```

Comment:
@detiber I didn't use the prompt=True because of the current way that we print the values and the way the values are needed to populdate the master.

When it requests the var it performs as follows

```
Github organization [()]: openshift
	github_organization: o,p,e,n,s,h,i,f,t
Continue using these values? [y/N]: y
```

```
ansible-playbook -i inventory/aws/hosts -e 'region=us-east-1     stack_name=openshift-infra     ami=ami-a33668b4     keypair=OSE-key     create_key=no     key_path=/dev/null     create_vpc=yes     vpc_id=None     private_subnet_id1=None     private_subnet_id2=None     private_subnet_id3=None     public_subnet_id1=None     public_subnet_id2=None     public_subnet_id3=None     byo_bastion=no     bastion_sg=/dev/null     master_instance_type=t2.medium     node_instance_type=t2.medium     app_instance_type=t2.medium     bastion_instance_type=t2.micro     public_hosted_zone=rcook-aws.sysdeseng.com     wildcard_zone=apps.rcook-aws.sysdeseng.com     console_port=443     deployment_type=openshift-enterprise     rhsm_user=sde-rcook     rhsm_password=4Wa12TbIs!     rhsm_pool="Red Hat OpenShift Container Platform, Premium, 2-Core"     containerized=False     s3_bucket_name=openshift-infra-ocp-registry-rcook-aws     s3_username=openshift-infra-s3-openshift-user     github_client_id=3a90715d36470ad14a9c     github_client_secret=afsd     github_organization=["s","a","f"]' playbooks/infrastructure.yaml
```